opengrok-indexer/src/main/java/org/opengrok/indexer/analysis/archive/GZIPAnalyzer.java

```diff
@@ -98,7 +98,7 @@ public void analyze(Document doc, StreamSource src, Writer xrefOut)
             }
             if (fa == null) {
                 this.g = Genre.DATA;
-                LOGGER.log(Level.WARNING, "Did not analyze {0}, detected as data.", newname);
+                LOGGER.log(Level.FINEST, "Did not analyze {0}, detected as data.", newname);
```

Comment:
There could be some new keywords here, like "did not."

Reply:
I will add.

Reply:
No. Please *think* about this not just blindly act on it.

Reply:
"does not", "doesn't" are keywords. "did not" is similar to them. I think it should be a keyword.